boards/arc/emsdp/doc/index.rst

```diff
@@ -0,0 +1,209 @@
+
+Configuring
+-----------
+
+You may need to write a prj_arc.conf file if the sample doesn't have one.
+Next, you can use the menuconfig rule to configure the target. By specifying
+``emsdp`` as the board configuration, you can select the ARC EM Software
+Development Platform board support for Zephyr.
+
+.. zephyr-app-commands::
+   :board: emsdp
+   :zephyr-app: samples/hello_world
+   :goals: menuconfig
+
+
+Building
+--------
+
+You can build an application in the usual way.  Refer to
+:ref:`build_an_application` for more details. Here is an example for
+:ref:`hello_world`.
+
+.. zephyr-app-commands::
+   :board: emsdp
+   :zephyr-app: samples/hello_world
+   :maybe-skip-config:
+   :goals: build
+
+Connecting Serial Output
+=========================
+
+In the default configuration, Zephyr's EM Software Development Platform images
+support serial output via the USB-UART on the board. To enable serial output:
+
+* Open a serial port emulator (i.e. on Linux minicom, putty, screen, etc)
+
+* Specify the tty driver name, for example, on Linux this may be
+  :file:`/dev/ttyUSB0`
+
+* Set the communication settings to:
+
+
+========= =====
+Parameter Value
+========= =====
+Baud:     115200
+Data:     8 bits
+Parity:    None
+Stopbits:  1
+========= =====
+
+Debugging
+==========
+
+Using the latest version of Zephyr SDK(>=0.9), you can debug and flash IoT
+Development Kit directly.
+
+One option is to build and debug the application using the usual
+Zephyr build system commands.
+
+.. zephyr-app-commands::
+   :board: emsdp
+   :app: <my app>
+   :goals: debug
+
+At this point you can do your normal debug session. Set breakpoints and then
+'c' to continue into the program.
+
+The other option is to launch a debug server, as follows.
+
+.. zephyr-app-commands::
+   :board: emsdp
+   :app: <my app>
+   :goals: debugserver
+
+Then connect to the debug server at the EM Software Development Platform from a
+second console, from the build directory containing the output :file:`zephyr.elf`.
+
+.. code-block:: console
+
+   $ cd <my app>
+   $ $ZEPHYR_SDK_INSTALL_DIR/sysroots/x86_64-pokysdk-linux/usr/bin/ \
+      arc-zephyr-elf/arc-zephyr-elf-gdb zephyr.elf
+   (gdb) target remote localhost:3333
+   (gdb) load
+   (gdb) b main
+   (gdb) c
+
+Flashing
+========
+
+If you just want to download the application to the EM Software Development
+Platform's CCM and run, you can do so in the usual way.
+
+.. zephyr-app-commands::
+   :board: emsdp
+   :app: <my app>
+   :goals: flash
+
+This command still uses openocd and gdb to load the application elf file to EM
+Software Development Platform, but it will load the application and immediately run.
+If power is removed, the application will be lost since it wasn't written to flash.
+
+Most of the time you will not be flashing your program but will instead debug
+it using openocd and gdb. The program can be download via the USB cable into
+the code and data memories.
+
+Release Notes
+*************
+
+References
```

Comment:
Same here (a heading with no content).   If you want the links below to actually show up in the output here, add a ``target-notes`` directive before the first link:
```
References
**********
.. target-notes::

.. _Digilent Pmod Modules: http://store.digilentinc.com/pmod-modules
.. _Putty website: http://www.putty.org
```

Reply:
added